File: Python/Best Time to Buy and Sell Stock II.py
class Solution(object):
    def maxProfit(self, prices):
        """
        :type prices: List[int]
        :rtype: int
        """
        counter = 0
        for i in xrange(len(prices) - 1):
            if prices[i + 1] > prices[i]:
                counter += prices[i + 1] - prices[i]
        return counter

File: cpp/Best Time to Buy and Sell Stock II.cpp
class Solution {
public:
    int maxProfit(vector<int>& prices) {
        int counter = 0;
        for (int i = 0; i + 1 < prices.size(); ++i) {
            if (prices[i + 1] > prices[i]) {
                counter += (prices[i + 1] - prices[i]);
            }
        }
        return counter;
    }
};
